Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/GamePropertiesDialog.java

```diff
@@ -107,6 +108,9 @@ public Dialog onCreateDialog(Bundle savedInstanceState)
     itemsBuilder.add(R.string.properties_edit_cheats, (dialog, i) ->
             CheatsActivity.launch(getContext(), gameId, gameTdbId, revision, isWii));
 
+    itemsBuilder.add(R.string.properties_ini_editor, (dialog, i) ->
+            IniEditorDialog.newInstance(path).show(getFragmentManager(), "IniEditorDialog"));
```

Comment:
`newInstance` wants a game ID, but you're giving it a path.

Reply:
A NullPointerException happens if I do that

Reply:
Oh. Yeah, I see what's wrong now. In IniEditorDialog.onCreateDialog, you're doing `GameFileCacheManager.addOrGet(getArguments().getString(ARG_GAME_ID))`, which doesn't make sense. Just do `final String gameId = getArguments().getString(ARG_GAME_ID);`.

Reply:
fixed, thanks